cmd/import.go

```diff
@@ -111,6 +116,41 @@ func (cmd *ImportDataCmd) handleCsv(ctx context.Context, infoSchema *spanner.Inf
 
 }
 
+func (cmd *ImportDataCmd) handleDatabaseDumpFile(ctx context.Context, dbUri, sourceFormat string, dialect string, spannerAccessor spanneraccessor.SpannerAccessor) error {
+
+	importDump, err := import_file.NewImportFromDump(cmd.project, cmd.instanceId, cmd.databaseName, cmd.sourceUri, sourceFormat, spannerAccessor)
+	if err != nil {
+		return err
+	}
+
+	defer importDump.Finalize()
+
+	schemaStartTime := time.Now()
+	conv, err := importDump.CreateSchema(dialect)
+	if err != nil {
+		return fmt.Errorf(fmt.Sprintf("can't create schema: %v\n", err))
+	}
+
+	// TODO: Only update database
+	err = spannerAccessor.CreateOrUpdateDatabase(ctx, dbUri, sourceFormat, conv, sourceFormat)
+	spannerAccessor.Refresh(ctx, dbUri)
+
+	schemaEndTime := time.Now()
+	elapsedTime := schemaEndTime.Sub(schemaStartTime)
+	logger.Log.Info(fmt.Sprintf("Schema creation took %f secs", elapsedTime.Seconds()))
+	if err != nil {
+		return err
+	}
+
+	err = importDump.ImportData(conv)
+
+	dataEndTime := time.Now()
+	elapsedTime = dataEndTime.Sub(schemaEndTime)
+	logger.Log.Info(fmt.Sprintf("Data import took %f secs", elapsedTime.Seconds()))
```

Comment:
Trying to recall, is there some sort of a progress bar that appears on the screen when import is in progress? This will be important from a UX perspective.

Same comment applies to schema as well - Does schema creation take long? I feel like that there should be some notion of progress (like a timer or something) that should denote long-running operations so that the user knows that they are not stuck.

Reply:
From UX perspective in:
CLI: there are logs which depict the progress.

WEB Ui: this will be thought through when UI integration is done.

Reply:
Can you show me a screenshot of what the logs look like? Yes, web UI is not a concern right now. Only asking for UX of the CLI.

Reply:
https://screenshot.googleplex.com/9hKYBhMjZBzdqdZ

Reply:
As discussed over chat, we have created a follow up bug on this.